Sections/Rotary.cs

```diff
@@ -0,0 +1,153 @@
+using SimHub.Plugins;
+
+namespace DahlDesign.Plugin.Categories
+{
+    /// <summary>
+    /// Rotary button class
+    /// </summary>
+    public class Rotary : SectionBase
+    {
+
+        public Rotary(DahlDesign dahlDesign) : base(dahlDesign) { }
```

Comment:
When using increments/decrements the menu goes 1-12, but using any of the specific bindings brings it to 13, and it is locked there.

Reply:
Ok, fixed in the latest commit. Don't know why that fixed it thb, might be something about how these actions are built?